spec/music_album_spec.rb

```diff
@@ -0,0 +1,16 @@
+require_relative '../classes/music_album'
+
+describe MusicAlbum do
+  context 'It should create a musicAlbum instance with onspotify, name and published date' do
+    music = MusicAlbum.new(publish_date: '2012-05-01', name: true, on_spotify: 'Hello')
```

Comment:
Please observe the correct argument types. on_spotify is a boolean, and publish_date is a Date from the DateTime class.
Incorrect arguments lead to invalid tests that don't simulate the behaviour of the code. This defeats the purpose of testing.